source/documentation/standards/how-to-get-a-domain-name.html.md.erb

```diff
@@ -0,0 +1,53 @@
+---
+title: How to get, register or manage a domain name
+last_reviewed_on: 2021-09-16
+review_in: 3 months
+---
+
+# <%= current_page.data.title %>
+
+The MOJ centrally registers and maintains domains for use across the department.
+Domains used within the MOJ must follow the [naming domains]
+(naming-domains.html) standards and must be registered by the [MOJ's Domains
+team](mailto:domains@digital.justice.gov.uk).
+
+As per the [Naming domains guidance]
+(naming-domains.html#non-gov-uk-domains), there are various reasons why you
+shouldn't use a non-`gov.uk` domain name for your service.
```

Comment:
I wonder if this could be rephrased to not use a double negative. Only because I had to read it twice, got the opposite meaning on first read.

Something like:
As per the [Naming domains guidance]
(naming-domains.html#non-gov-uk-domains), the use of non-`gov.uk` domain names is strongly discouraged.

Reply:
Thanks! I've replaced with your suggestion.